### DISCUSSION TITLE
"This note has no real or implied parents"

### DISCUSSION THREAD

**Discussion by oyvindsolstad** [SkepticMystic/breadcrumbs#337] [Q&A]
Is there a way to turn OFF this warning on every note? 

**"This note has no real or implied parents"**

So annoying, that I have stopped using the plugin.

**Comment by SkepticMystic** [comment]
You can disable it by changing the No Path Found Message to be blank

**Reply by oyvindsolstad** [reply]
Thanks!

**Comment by michaelpporter** [comment]
Good news — in v4 this message is **off by default**, so it no longer shows up on every parentless note.

It's controlled by the **No path message** setting under **Settings → Page Views → Trail**. That field now defaults to empty, and the Trail view only renders a message when the field is non-empty. So out of the box you'll see nothing on notes with no parents.

If you (or an imported v3 config) ever set it and want it gone again, just clear that field. 🍞
